package.json

```diff
@@ -69,5 +72,12 @@
     "plugins": [
       "transform-class-properties"
     ]
+  },
+  "lint-staged": {
+    "src/**/*.js": [
+      "prettier --print-width=120 --single-quote --trailing-comma=all --write",
```

Comment:
I like prettier, but is there a way to take out "print-width"? Modern tooling should not care about line lengths. And if you are a vim or emacs user, you are such an incredible power user that you already know how to make those present virtual line breaks without actually saving them into the actual files.